README.md

```diff
@@ -120,6 +120,19 @@ VARIABLE | REQUIRED | TYPE | DESCRIPTION
 [get groups](#action-get-groups) - Gets a list of groups  
 [add group user](#action-add-group-user) - Add user to group  
 [remove group user](#action-remove-group-user) - Remove user from group  
+[get whitelist](#action-get-whitelist) - Get urls on the allow list  
```

Comment:
Any reason we're using _both_ allowlist/denylist and whitelist/blacklist terminology? Should we standardize on one?

Reply:
in the new epic we were using whitelist/blacklist, but it seems like zscaler prefers allowlist/denylist. will standardize to one.

Reply:
I believe Splunk prefers allowlist/denylist as well